_posts/2017-07-14-tutorial1-ritotjs-basic-component.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title:  "Riot Tutorial 1 - Componente Básico"
+date:   2017-07-14 10:00:00 -0300
+tags: tutorial
+categories: en
+comments: true
+---
+
+En este tutorial vamos a crear un componente o etiqueta personalizada
+utilizando [riot.js](http://riotjs.com/){:target="\_blank"} el cuál mostrará un
+mensaje de saludo con un nombre previamente establecido, o con la palabra
+"mundo" sino se suministra ningún nombre.
+
+## Componente
+Los componentes **pueden incluír** tres secciones:
+1. **CSS** (estilos del componente).
+2. **HTML**.
+3. **Javascript** (comportamiento deseado).
+
+
+```html
+<!-- hello-world.tag -->
+<hello-world>
+  <style>
+    h1 { color: red; }
+  </style>
+
+  <h1>hola {name}!</h1>
+
+  <script>
+    name = opts.name
```

Comment:
esto sería `this.name = opts.name || "mundo"` de esta forma es más simple el montaje del elemento (y es más fácil de utilizar).